src/GlobalStylesheets.js

```diff
@@ -55,13 +51,15 @@ function reap() {
 }
 
 var GlobalStylesheets = {
-  install: function() {
+  install: function(config = {}) {
+    validateConfig(config);
+    GlobalStylesheets.injection = assign(defaultConfig, config);
     if (browser) {
       setInterval(reap, 10000);
     }
   },
 
-  getKey: function(styleObj, displayName, component) {
+  getKey: function(styleObj, name) {
```

Comment:
Changing this to name probably wasn't the smartest thing.  Would be nice if this was the parent component's displayName.  (e.g. `Block` is not as meaningful as `Avatar`)